projects/common/src/lib/elements/manage/manage.component.html

```diff
@@ -120,15 +120,24 @@ <h2>Add a New Device</h2>
         <div class="telemetry widget">
           <mat-card>
             <mat-card-header>
-              <mat-card-title>Device Telemetry</mat-card-title>
+              <div  fxLayout="column" fxLayoutAlign="start start">
+                <mat-card-title>Device Telemetry</mat-card-title>
+                <lcu-refresh-select 
+                  [state]="State"
+                  (refresh-rate-changed)="RefreshRateChanged($event)">
+                </lcu-refresh-select>  
+              </div>
 
               <div fxFlex></div>
+              <div fxLayout="column" fxLayoutAlign="start end">
+                <lcu-enabled-toggle
+                  [enabled]="State.Telemetry?.Enabled"
+                  enabled-text="Live"
+                  (enabled-toggled)="ToggleTelemetryEnabledChanged($event)"
+                ></lcu-enabled-toggle>
+                <div id="last-sync-text"*ngIf="LastSyncedAt && State.Telemetry?.Enabled">Last Updated: {{LastSyncedAt}}</div>

```

Comment:
Instead of an id, i think a class="last-sync-text" is more in line with what we have elsewhere, and there should be a space between ...text"*ngif...

Also, on the div, just use our spread class instead of setting margin in the scss:
<div class="last-sync-text spread" ....